### PR TITLE
singlesourced most examples

### PR DESCRIPTION
**Pull request by eneufeld**
Only the Live example and the rule example are not yet singlesourced

### CODE REVIEW

**Comment by eneufeld** [comment]
@edgarmueller could you review and publish?

File: src/app.js [hunk 1]
```diff
@@ -586,5 +588,5 @@ require('./examples/live-edit.controller');
 require('./examples/generate-ui.controller');
 require('./examples/generate-schema.controller');
 require('./examples/custom.controller');
-require('./examples/custom.jsf');
+// require('./examples/custom.jsf');
```
Comment (edgarmueller) [inline src/app.js]: Remove comment

File: src/examples/categories.controller.js [hunk 2]
```diff
@@ -1,81 +1,9 @@
...
-            vegetarian: false,
-            birthDate: "1985-06-02"
-        };
-
-        vm.formattedData = function() {
-            return JSON.stringify(vm.data, null, 4);
-        };
-    });
+    .controller('CategoriesController',['category.schema','category.uischema','category.data', function(Schema,UISchema,Data) {
+    var vm = this;
```
Comment (edgarmueller) [inline src/examples/categories.controller.js]: Fix identation. Unfortunately we didn't setup jslint for the website (yet).

File: src/examples/remote-ref.controller.js [hunk 3]
```diff
@@ -1,69 +1,9 @@
...
-        vm.data = {
-
-        };
-
-        vm.formattedData = function() {
-            return JSON.stringify(vm.data, null, 4);
-        };
-    });
+    .controller('RemoteRefController', ['resolve.schema','resolve.uischema','resolve.data', function(Schema,UISchema,Data) {
+    var vm = this;
```
Comment (edgarmueller) [inline src/examples/remote-ref.controller.js]: Fix identation

File: src/examples/rule.controller.js [hunk 4]
```diff
@@ -1,5 +1,17 @@
 'use strict';
+//TODO: Replace with this:
```
Comment (edgarmueller) [inline src/examples/rule.controller.js]: Remove comment

**Comment by eneufeld** [comment]
the comment in the rule controller is intended as a reminder

**Pull request merged**